debian/libostree-1-1.symbols

```diff
@@ -150,6 +150,7 @@ libostree-1.so.1 libostree-1-1 #MINVER#
  ostree_remote_unref@LIBOSTREE_2017.6_EXPERIMENTAL 2017.8
  ostree_remote_get_name@LIBOSTREE_2017.7_EXPERIMENTAL 2017.8
  ostree_remote_get_type@LIBOSTREE_2017.14_EXPERIMENTAL 2017.12-0endless3
+ ostree_remote_get_url@LIBOSTREE_2017.14_EXPERIMENTAL 2017.12-0endless4
```

Comment:
I’d make the last bit be `2017.12-0endless5`, and add a new entry to `debian/changelog` to increment the version number to that, so you can depend on this version of ostree.deb in flatpak, and guarantee the new API is present for you to use.

i.e. Like what Dan did in commit 30dbc634cd3fd9bbe64fe2f91d21641bac2c730e.